requirements-provider.txt

```diff
@@ -2,13 +2,17 @@ elasticsearch<8
 elasticsearch-dsl<8
 fiona
 #GDAL>=3.0.0
+geoalchemy
 netCDF4
 pandas; python_version < '3.7'
 pandas==1.2.5; python_version >= '3.7'
-psycopg2
+psycopg2-binary
+pygeofilter[backend-sqlalchemy]
 pygeometa
 pymongo==3.10.1
 scipy
+shapely
```

Comment:
Shapely is already included in `requirements.txt`

Reply:
I removed it now.  And added `pygeofilter` to `requirements.txt`.

I couldn't find a `pygeofilter` package for `debian/control`. https://packages.debian.org/search?keywords=pygeofilter&searchon=names&suite=stable&section=all

Reply:
cc'ing @kalxas here for comments w.r.t. pygeofilter packaging